packages/esm-appointments-app/src/appointments/forms/cancel-form/cancel-appointment.component.tsx

```diff
@@ -38,8 +38,8 @@ const CancelAppointment: React.FC<CancelAppointmentProps> = ({ appointment }) =>
         subtitle: t('cancelledSuccessfully', 'It has been cancelled successfully'),
         title: t('appointmentCancelled', 'Appointment cancelled'),
       });
-      mutate(`${restBaseUrl}/appointment/appointmentStatus?forDate=${currentAppointmentDate}&status=Scheduled`);
-      mutate(`${restBaseUrl}/appointment/appointmentStatus?forDate=${currentAppointmentDate}&status=Cancelled`);
+      mutate(`${restBaseUrl}/appointment/appointmentStatus?forDate=${selectedDate}&status=Scheduled`);
+      mutate(`${restBaseUrl}/appointment/appointmentStatus?forDate=${selectedDate}&status=Cancelled`);
```

Comment:
Not germane to this PR, but the implementation here should probably be something closer to:

```ts
mutate(key =>
  typeof key === 'string' && key.startswith(`${restBaseUrl}/appointment/appointmentStatus?forDate=${selectedDate}`),
  undefined,
  { revalidate: true }
);
```

That way, it's agnostic as to status...